config.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget android-versionCode="20000854" id="org.jellyfin.mobile" version="2.8.54" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+<widget xmlns="http://www.w3.org/ns/widgets"
+    xmlns:cdv="http://cordova.apache.org/ns/1.0"
+    id="org.jellyfin.mobile"
+    version="2.8.54"
+    android-versionCode="20000854">
```

Comment:
I have found Cordova will overwrite `config.xml` and remove formatting like this. I prefer the multiline formatting here, but I don't think Cordova will keep it.

Reply:
It seems to be keeping it in the root directory, just not in the platforms directory.

Reply:
This may have only happened when changes to platforms are made. I'm fine with keeping it for now, but it may need reverted once we are able to remove the platforms directory.